tools/genesis-snapshot/presets/presets.go

```diff
@@ -15,238 +15,242 @@ import (
+				Ed25519 Public Key:   5b938327ea463e0c323c0fd44f6fc1843ed94daecc6909c6043d06b7152e4737
+				Ed25519 Address:      rms1qqhkf7w30xv375z59vq7qd86qsa3j4qrsadcval04uvkkswg3qpaqf4hga2
+				Restricted Address:   rms1xqqz7e8e69uej86s2s4srcp5lgzrkx25qwr4hpnha7h3j66pezyq85qpqg55v3ur, Capabilities: mana
 			*/
-			AccountID:            blake2b.Sum256(lo.PanicOnErr(hexutil.DecodeHex("0x01fb6b9db5d96240aef00bc950d1c67a6494513f6d7cf784e57b4972b96ab2fe"))),
-			Address:              iotago.Ed25519AddressFromPubKey(lo.PanicOnErr(hexutil.DecodeHex("0x01fb6b9db5d96240aef00bc950d1c67a6494513f6d7cf784e57b4972b96ab2fe"))),
-			Amount:               mock.MinValidatorAccountAmount,
-			IssuerKey:            iotago.Ed25519PublicKeyBlockIssuerKeyFromPublicKey(ed25519.PublicKey(lo.PanicOnErr(hexutil.DecodeHex("0x01fb6b9db5d96240aef00bc950d1c67a6494513f6d7cf784e57b4972b96ab2fe")))),
-			ExpirySlot:           iotago.MaxSlotIndex,
-			BlockIssuanceCredits: iotago.MaxBlockIssuanceCredits / 4,
-			StakingEpochEnd:      iotago.MaxEpochIndex,
-			FixedCost:            1,
-			StakedAmount:         mock.MinValidatorAccountAmount,
-			Mana:                 iotago.Mana(mock.MinValidatorAccountAmount),
-		},
-		snapshotcreator.AccountDetails{
-			/*
-				node-02-validator
+			snapshotcreator.BasicOutputDetails{
+				Address: lo.Return2(iotago.ParseBech32("rms1xqqz7e8e69uej86s2s4srcp5lgzrkx25qwr4hpnha7h3j66pezyq85qpqg55v3ur")),
+				Amount:  1_000_000_000_000_000,
+				Mana:    10_000_000,
+			},
+		),
+	}
 
-				Ed25519 Public Key:   83e7f71a440afd48981a8b4684ddae24434b7182ce5c47cfb56ac528525fd4b6
-				Ed25519 Address:      rms1qzjwamvhjuqtw3dkfwmmj2fgcetcdyt4uxrnjxel4caxfstzz903ypu8xvn
-				Account Address:      rms1pzjwamvhjuqtw3dkfwmmj2fgcetcdyt4uxrnjxel4caxfstzz903y7hhr3d
-				Restricted Address:   rms1xqq2fmhdj7tspd69ke9m0wff9rr90p53whscwwgm87hr5expvgg47yspqgm6whkx, Capabilities: mana
-			*/
-			AccountID:            blake2b.Sum256(lo.PanicOnErr(hexutil.DecodeHex("0x83e7f71a440afd48981a8b4684ddae24434b7182ce5c47cfb56ac528525fd4b6"))),
-			Address:              iotago.Ed25519AddressFromPubKey(lo.PanicOnErr(hexutil.DecodeHex("0x83e7f71a440afd48981a8b4684ddae24434b7182ce5c47cfb56ac528525fd4b6"))),
-			Amount:               mock.MinValidatorAccountAmount,
-			IssuerKey:            iotago.Ed25519PublicKeyBlockIssuerKeyFromPublicKey(ed25519.PublicKey(lo.PanicOnErr(hexutil.DecodeHex("0x83e7f71a440afd48981a8b4684ddae24434b7182ce5c47cfb56ac528525fd4b6")))),
-			ExpirySlot:           iotago.MaxSlotIndex,
-			BlockIssuanceCredits: iotago.MaxBlockIssuanceCredits / 4,
-			StakingEpochEnd:      iotago.MaxEpochIndex,
-			FixedCost:            1,
-			StakedAmount:         mock.MinValidatorAccountAmount,
-			Mana:                 iotago.Mana(mock.MinValidatorAccountAmount),
-		},
-		snapshotcreator.AccountDetails{
-			/*
-				node-03-validator
+	// Feature is a preset for the feature network, genesis time ~20th of July 2023.
```

Comment:
This sentence makes no sense, since we do `time.Now().Unix()-100_000`

Reply:
fixed